net/qpid-proton/Portfile

```diff
@@ -1,7 +1,12 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
-PortGroup           cxx11 1.1
+
+PortGroup           github 1.0
+PortGroup           cmake 1.1
+
+github.setup        apache qpid-proton 0.31.0
+revision            0
 
 name                qpid-proton
```

Comment:
The `name` line is redundant here and is set to this by the `github.setup` command.

Reply:
@mf2k good to know! Made the update to remove the name and repushed - will monitor.